src/main/resources/templates/user/create.html

```diff
@@ -96,7 +98,6 @@
                                     <option value="+48">+48 (Poland)</option>
                                     <option value="+65">+65 (Singapore)</option>
                                     <option value="+60">+60 (Malaysia)</option>
-                                    <option value="+34">+34 (Spain)</option>
                                     <option value="+356">+356 (Malta)</option>
```

Comment:
Any reason to delete Spain from country codes?

Reply:
NO sir , it was a mistake to remove Spain and Indonesia from the country codes.

Reply:
> NO sir , it was a mistake to remove Spain and Indonesia from the country codes.

Thanks for confirming